Missing "npx" dependency

On a stock Ubuntu 18.04 machine installing `nodejs` and `npm` running tests fails with an error about a missing npx command.

Installing `npx` from apt fixes the problem. I am not a Node person and I'm not really sure where this dependency should be listed so passing along FYI.

```
$> npm test
> pelias-placeholder@0.0.0-development test /home/ubuntu/placeholder
> npm run units


> pelias-placeholder@0.0.0-development units /home/ubuntu/placeholder
> ./cmd/units

./cmd/units: line 7: npx: command not found

npm ERR! Linux 4.15.0-1044-aws
npm ERR! argv "/usr/bin/node" "/usr/bin/npm" "run" "units"
npm ERR! node v8.10.0
npm ERR! npm  v3.5.2
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! pelias-placeholder@0.0.0-development units: `./cmd/units`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the pelias-placeholder@0.0.0-development units script './cmd/units'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the pelias-placeholder package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     ./cmd/units
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs pelias-placeholder
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls pelias-placeholder
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ubuntu/placeholder/npm-debug.log
npm ERR! Test failed.  See above for more details.
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,7 +3,6 @@ language: node_js
 notifications:
   email: false
 node_js:
-  - 8
   - 10
   - 12
 matrix:

--- a/package.json
+++ b/package.json
@@ -2,7 +2,7 @@
   "name": "pelias-placeholder",
   "version": "0.0.0-development",
   "engines": {
-    "node": ">=8.0.0"
+    "node": ">=10.0.0"
   },
   "author": "mapzen",
   "license": "MIT",

